.eslintrc.js

```diff
@@ -58,6 +58,12 @@ module.exports = {
     '@typescript-eslint/no-explicit-any': 'error',
     camelcase: 'off',
     'no-unused-vars': 'off', // Disable the base rule it can report incorrect errors
+    'no-unused-expressions': [
+      'error',
+      {
+        allowShortCircuit: true,
```

Comment:
🤔 Where did this rule break?

Reply:
The original code I wrote was doing something with possible nonexistent event handlers. Not sure what the reason was I couldn't do the optional call like I am now. Looks like it doesn't break anymore so I will remove it.